app/services/get-worker-for-task.js

```diff
@@ -7,6 +7,7 @@ var sendClaimNotification = require('./workers/send-claim-notification')
 var completeClaim = require('./workers/complete-claim')
 var requestInformationResponse = require('./workers/request-information-response')
 var dwpCheck = require('./workers/dwp-check')
+var sendPerformancePlatformMetricsForDay = require('./workers/send-performance-platform-metrics-for-day')
```

Comment:
Send vs post - `send-performance-platform-metrics-for-day` used for worker and `post-performance-platform-metrics-for-day` used for service. Should we only use one?